iron-overlay-backdrop.html

```diff
@@ -0,0 +1,101 @@
+<!--
+@license
+Copyright (c) 2015 The Polymer Project Authors. All rights reserved.
+This code may only be used under the BSD style license found at http://polymer.github.io/LICENSE.txt
+The complete set of authors may be found at http://polymer.github.io/AUTHORS.txt
+The complete set of contributors may be found at http://polymer.github.io/CONTRIBUTORS.txt
+Code distributed by Google as part of the polymer project is also
+subject to an additional IP rights grant found at http://polymer.github.io/PATENTS.txt
+-->
+
+<link rel="import" href="../polymer/polymer.html">
+
+<link rel="import" href="iron-overlay-manager.html">
+
+<style is="x-style">
+
+  * {
+    --iron-overlay-backdrop-background-color: #000;
+  }
+
+</style>
+
+<dom-module id="iron-overlay-backdrop">
+
+  <style>
+
+    :host {
+      position: fixed;
+      top: 0;
+      left: 0;
+      width: 100vw;
+      height: 100vh;
+      background-color: var(--iron-overlay-backdrop-background-color);
+      opacity: 0;
+      transition: opacity 0.2s;
+
+      mixin(--iron-overlay-backdrop);
+    }
+
+    :host(.iron-opened) {
+      opacity: 0.6;
+
+      mixin(--iron-overlay-backdrop-opened);
+    }
+
+  </style>
+
+</dom-module>
+
+<script>
+
+(function() {
+
+  Polymer({
+
+    is: 'iron-overlay-backdrop',
+
+    enableCustomStyleProperties: true,
+
+    properties: {
+
+      _manager: {
+        type: Object,
+        value: Polymer.IronOverlayManager
+      }
+
+    },
+
+    prepare: function() {
+      if (!this.parentNode) {
+        Polymer.dom(document.body).appendChild(this);
+        this.style.zIndex = this._manager.currentOverlayZ() - 1;
+      }
+    },
+
+    open: function() {
+      // only need to make the backdrop visible if this is called by the first overlay with a backdrop
+      if (this._manager.getBackdrops().length < 2) {
+        this.classList.add('iron-opened');
```

Comment:
Since the backdrop is its own component now, using an attribute seems to make more sense. Fixed.